react/components/RadioGroup/index.js

```diff
@@ -44,8 +47,10 @@ class RadioGroup extends React.Component {
             const id = `${name}-${i}`
             return (
               <label
-                className={`db br3 ${classNames({
-                  'b--muted-4 ba pv2 ph4': !hideBorder,
+                className={`${classNames('db br3', {
+                  'ba pv2 ph4': !hideBorder,
+                  'b--danger hover-b--danger': errorHighlight,
+                  'b--muted-4': !error && !errorMessage,
                   pointer: !isDisabled,
                 })}`}
```

Comment:
😅 
```suggestion
                className={classNames('db br3', {
                  'ba pv2 ph4': !hideBorder,
                  'b--danger hover-b--danger': errorHighlight,
                  'b--muted-4': !error && !errorMessage,
                  pointer: !isDisabled,
                })}
```

Reply:
I had tried without this and kind of broke the styles. But I have tested now and it works perfectly, that's strange hahaha but I will change to this.